src/components/tickets/ticket/create-ticket/create-ticket.tsx

```diff
@@ -67,159 +72,144 @@ export class CreateNewTicket extends React.Component<
     description: '',
     setReminder: false,
     type: this.props.type ?? 'REMIND',
+    currentUser: {
+      iEXOptionsIndex: 0,
+      email: null,
+    },
```

Comment:
I am not sure how the options are set, either this should be 0 as it is, or -1 so the `find` below (which I am pretty sure returns -1 if it is not found) behaves in the same way as the defualt.
```suggestion
    currentUser: {
      iEXOptionsIndex: -1,
      email: null,
    },
```

Reply:
iEXOptionsIndex = 0 defaults to "Unassigned" in iEXOptions which might be a preferable default